docs/source/pillars/information_governance.md

```diff
@@ -156,64 +295,200 @@ _The ability of the TRE organisation to manage projects effectively._
   - Mandatory
 ```
 
-### Roles and responsibilities
+**Study Management Portal:** An online platform that provides centralised access to manage research studies including onboarding studies, control of access and administration of compliance tasks.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-roles-responsibilities
+:name: tab-ig-study-management-portal
+
 * - Statement
   - Guidance
   - Importance
-* - You must have clearly defined roles and responsibilities for all operators and users of your TRE.
-  - This may include roles such as users, system administrators, system operators or data providers.
-    Every member of your TRE organisation should have at least one pre-defined role with clear powers and responsibilities.
-  - Mandatory
+* - You could implement a portal that can provide a workflow engine and database which automates the processes within this capability.
+  - A portal should automate as much of the processes within the capability as possible.
+    Where processes are automated process maturity is easier to achieve with processes being completed more consistently and producing quality control and monitoring data automatically
+  - Optional
 ```
 
-## Member accreditation
+**Data Asset Register:** A database or other electronic record that documents and manages information about the TRE organisation's data assets, including their characteristics, ownership, usage, and other relevant details.
 
-_The ability of the TRE organisation to ensure that people with access to data are identified correctly, and they are suitably qualified._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-data-asset-register
 
-### Onboarding members
+* - Statement
+  - Guidance
+  - Importance
+* - You must keep a complete record of all the data assets held within the system.
+  - All data asset (current and past) held by the system should be retained along with meta-data useful for ensuring compliance can be demonstrated.
+    This would include ownership, data lifecycle, contracts, risk assessments and other quality data.
+    This is likely to already exist within the wider organisation but may require augmenting for the TRE.
+  - Mandatory
+```
+
+**Study Register:** A centralised record or database that tracks and manages information about research studies and projects.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-onboarding-members
+:name: tab-ig-study-register
 
 * - Statement
   - Guidance
   - Importance
-* - You must have clear onboarding processes in place for all roles within your TRE organisation.
-  - This may include all members signing role-specific terms of use, and completing role specific training.
+* - You must keep a complete record of all the research studies and projects within the TRE current and past.
+  - The study register should contain all data related to a study including a reference to data assets, members (researchers, owners etc.) and any compliance activities required.
   - Mandatory
```

Comment:
Making this mandatory implies that if you ever fail to do this for a single project, you can never be a TRE.

Reply:
Are you objecting to the statement being mandatory, or the guidance?

Reply:
Not sure why this is associated with the `onboarding processes` statement but it's meant to be the `study-register`. I'm objecting to making it mandatory to keep a complete record of all past, current and future projects. Records might be missing by accident (eg. legacy projects from before SATRE was adopted), or deliberately (eg. for confidentiality reasons) but I don't think this should cause you to fail your SATRE assessement.

Happy with this text at "Recommended" though.